packages/react/src/global-alert/GlobalAlert.test.tsx

```diff
@@ -61,7 +61,7 @@ describe('GlobalAlert', () => {
 			const { container } = renderGlobalAlert({
 				title: 'Alert title',
 				children: <Text as="p">Alert description.</Text>,
-				onDismiss: jest.fn(),
+				onClose: jest.fn(),
```

Comment:
not sure if I should add tests for the deprecated prop or not

Reply:
Could go either way. Probably since they'll still be in use a lot

Reply:
Unit tests updated to include onDismiss tests